CONTRIBUTING.md

```diff
@@ -15,6 +17,51 @@ This project follows the [Jupyter Code of Conduct].
 > - work as a team to get the pull requests merged
 > - show progress towards an accessibility _Roadmap_
 
+## Community Structure - The Jupyter Project and Subprojects
+The work being done in this repository will begin by centering around advocating for accessibility and building accessiblility-centered features within the JupterLab, JupyterNotebook, and JupyterHub sub-projects within the larger Jupyter project. Some information to connect you to those projects and organizations is located below.
+### The Jupyter Project
+* [Jupyter Website]
+* [Jupyter Gitter]
+
+- [Check out the accessibility section of the community forum](https://discourse.jupyter.org/c/special-topics/accessibility) for ongoing conversations and brainstorms around accessibility in the Jupyter ecosystem.
+- If you find something that you think should be labeled for accessibility, feel free to label it or add a comment.
+
+
+### JupyterLab
+[JupyterLab](http://jupyterlab.readthedocs.io/en/stable/) is the next-generation user interface for [Project Jupyter](https://jupyter.org) offering
+all the familiar building blocks of the classic Jupyter Notebook (notebook,
+terminal, text editor, file browser, rich outputs, etc.) in a flexible and
+powerful user interface.
+JupyterLab will eventually replace the classic Jupyter Notebook.
+* [JupyterLab Repository]
```

Comment:
Are these supposed to be links too?

Reply:
yes, similar to above. Reference style links with the references on line 180.
```
[JupyterLab Repository]:https://github.com/jupyterlab/jupyterlab
```

Reply:
Nope, links are good! I just was not familiar with the styling.